failed to create 2to1 bridge for topic '/rosout' with ROS 2 type

When excute "ros2 run ros1_bridge dynamic_bridge", terminal diaplay "failed to create 2to1 bridge for topic '/rosout' with ROS 2 type 'rcl_interfaces/Log' and ROS 1 type 'rosgraph_msgs/Log': No template specialization for the pair".

Hi, 

@dirk-thomas  @gonzodepedro I have a draft for this on https://github.com/aws-robotics/ros1_bridge/commits/issue-159-py, so for only adds selection to ROS 1 fields, but it would be easy to add it to ROS 2 fields too. However I'm having problems testing this, I cannot use FastRTPS due to https://github.com/ros2/rcl_interfaces/issues/61, and I wasn't able to build this for OpenSplice or RTI Connext. For example for RTI Connext I get the following error when building the bridge on a container for `osrf/ros2:nightly`

```
source /opt/rti.com/rti_connext_dds-5.3.1/setenv_ros2rti.bash
source /opt/ros/melodic/setup.bash
source /opt/ros/crystal/setup.bash
rm -rf build/ros1_bridge install/ros1_bridge/
colcon build --symlink-install --packages-up-to  ros1_bridge --cmake-force-configure
...
Finished <<< rosidl_typesupport_cpp [1.21s]                                                                          
--- stderr: rosidl_generator_py                             
CMake Error at cmake/rosidl_generator_py_generate_interfaces.cmake:317 (add_dependencies):
  The dependency target
  "rosidl_generator_py_custom__rosidl_typesupport_opensplice_c" of target
  "rosidl_generator_py__rosidl_typesupport_opensplice_c__pyext" does not
  exist.
Call Stack (most recent call first):
  /opt/workspace/install/ament_cmake_core/share/ament_cmake_core/cmake/core/ament_execute_extensions.cmake:38 (include)
  /opt/workspace/install/rosidl_cmake/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:280 (ament_execute_extensions)
  CMakeLists.txt:68 (rosidl_generate_interfaces)


---
Failed   <<< rosidl_generator_py	[ Exited with code 1 ]

Summary: 86 packages finished [43.7s]
  1 package failed: rosidl_generator_py
  4 packages had stderr output: rmw_opensplice_cpp rosidl_generator_py rosidl_typesupport_opensplice_c rosidl_typesupport_opensplice_cpp
  37 packages not processed
root@ub80c134adc2254:/opt/workspace# 
```

Any help would be much appreciated

--- a/rcl_interfaces/CMakeLists.txt
+++ b/rcl_interfaces/CMakeLists.txt
@@ -36,6 +36,8 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   DEPENDENCIES builtin_interfaces
 )
 
+install(FILES mapping_rules.yaml DESTINATION share/${PROJECT_NAME})
+
 # this must happen before the invocation of ament_package()
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)

--- a/rcl_interfaces/mapping_rules.yaml
+++ b/rcl_interfaces/mapping_rules.yaml
@@ -11,4 +11,3 @@
     file: 'file'
     function: 'function'
     line: 'line'
-  

--- a/rcl_interfaces/mapping_rules.yaml
+++ b/rcl_interfaces/mapping_rules.yaml
@@ -1,0 +1,14 @@
+-
+  ros1_package_name: 'rosgraph_msgs'
+  ros1_message_name: 'Log'
+  ros2_package_name: 'rcl_interfaces'
+  ros2_message_name: 'Log'
+  fields_1_to_2:
+    header.stamp: 'stamp'
+    level: 'level'
+    name: 'name'
+    msg: 'msg'
+    file: 'file'
+    function: 'function'
+    line: 'line'
+  

--- a/rcl_interfaces/package.xml
+++ b/rcl_interfaces/package.xml
@@ -24,5 +24,6 @@
 
   <export>
     <build_type>ament_cmake</build_type>
+     <ros1_bridge mapping_rules="mapping_rules.yaml"/>
   </export>
 </package>

